### PR TITLE
[native] Add TPC-DS Tests for Dwarf & Parquet

### PR DESCRIPTION
**Pull request by frankobe**
Added TestHiveTpcdsQueries to test results on 103 on both dwrf & parquet format.

In the sqls, decimal is cast to double
In the table creation, char(x) is cast to varchar

Signed-off-by: frankobe <mua08p@gmail.com>

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 {
     public TestHiveWindowQueries()
     {
-        super(true);
+        super(true, DWARF_STORAGE_FORMAT);
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java:23]: Spelling DWRF

File: presto-native-execution/src/test/java/com/facebook/presto/hive/HiveExternalWorkerQueryRunner.java [hunk 2]
```diff
@@ -462,6 +496,327 @@ private static void createBucketedLineitemAndOrders(QueryRunner queryRunner)
         }
     }
 
+    private static void createTpcdsCallCenter(QueryRunner queryRunner, Session session)
+    {
+        if (!queryRunner.tableExists(session, "call_center")) {
+            queryRunner.execute(session, "CREATE TABLE call_center AS " +
+                    "SELECT cc_call_center_sk, cast(cc_call_center_id as varchar) as cc_call_center_id, cast(cc_rec_start_date as varchar) as cc_rec_start_date, " +
```
[the remark this answers is not in the document]
Comment (frankobe) [reply presto-native-execution/src/test/java/com/facebook/presto/hive/HiveExternalWorkerQueryRunner.java:504]: @aditi-pandit do u know whether dwrf supports date?

[the remark this answers is not in the document]
Comment (aditi-pandit) [reply presto-native-execution/src/test/java/com/facebook/presto/hive/HiveExternalWorkerQueryRunner.java:504]: No. dwrf does not support date.

**Comment by v-jizhang** [comment]
@bot kick off tests

**Comment by isadikov** [comment]
I am following up on this.

**Comment by frankobe** [comment]
Close in favor of https://github.com/prestodb/presto/pull/19646

**Pull request closed without merging**